app/models/manageiq/providers/inventory/persister/builder/persister_helper.rb

```diff
@@ -21,16 +21,12 @@ module ManageIQ::Providers::Inventory::Persister::Builder::PersisterHelper
   #
   # @see documentation https://github.com/ManageIQ/guides/tree/master/providers/persister/inventory_collections.md
   #
-  def add_collection(builder_class, collection_name, extra_properties = {}, settings = {}, &block)
-    builder = builder_class.prepare_data(collection_name,
-                                         self.class,
-                                         make_builder_settings(settings),
-                                         &block)
+  def add_collection(builder_class, collection_name, collection_manager = nil, extra_properties = {}, settings = {}, &block)
```

Comment:
Note: this changes the interface to this method so we have to update any callers of `add_collection` that pass either extra_properties or settings

Reply:
So if I merge this now will it break things?

Reply:
Correct this needs a few other PRs to the providers and cross-repo tests